fsr.ino

```diff
@@ -611,6 +616,93 @@ class SerialProcessor {
     Serial.print("\n");
   }
 
+  const int16_t SAVE_SLOT_TAKEN_MARKER = -42;  // any negative number would do, as actual thresholds are >=0
```

Comment:
Also for code style, I would prefer to follow 80 character line limit (technically I use the Google Style Guide for this code: https://google.github.io/styleguide/cppguide.html)